### ISSUE TITLE
docker image is not starting

### ISSUE THREAD

**Issue by pulketo** [Kronuz/Xapiand#5]
Hello, idk if there is a problem with my docker install 
i'm running it con Centos 7 (3.10.0-862.14.4.el7.x86_64)
docker-ce-cli-18.09.1-3.el7.x86_64
docker-ce-18.09.1-3.el7.x86_64

docker ps show no running containers.
(firewalld is stopped)

**Comment by pulketo** [comment]
docker inspect 51253337df17
[
    {
        "Id": "51253337df17573a7ca4eac6d8b6123272bce5568d7929e0a45fc625020a5e82",
        "Created": "2019-01-15T15:52:31.256782054Z",
        "Path": "/entrypoint.sh",
        "Args": [
            "xapiand"
        ],
        "State": {
            "Status": "exited",
            "Running": false,
            "Paused": false,
            "Restarting": false,
            "OOMKilled": false,
            "Dead": false,
            "Pid": 0,
            "ExitCode": 1,
            "Error": "",
            "StartedAt": "2019-01-15T15:52:31.801772373Z",
            "FinishedAt": "2019-01-15T15:52:31.746725638Z"
..
..
..

**Comment by maciej-trebacz** [comment]
Same for me, `docker run -p 8880:8880 --rm dubalu/xapiand` returns almost immediately with no output.

**Comment by Kronuz** [comment]
There was a bug in the latest entrypoint. It should be fixed in the upcoming v0.8.12 (to be ready in a couple hours).

**Comment by Kronuz** [comment]
Done! The following should work:

```sh
docker pull dubalu/xapiand:latest
docker run -p 8880:8880 --rm dubalu/xapiand -vvvv --colors
```
